src/test/scala/fairysf/breakthroughtroyka/MiniBreakthroughTroykaFenTest.scala

```diff
@@ -0,0 +1,16 @@
+package strategygames.fairysf
+
+import org.specs2.matcher.ValidatedMatchers
+import org.specs2.mutable.Specification
+import strategygames.fairysf.variant.MiniBreakthroughTroyka
+
+class MiniBreakthroughTroykaFenTest extends Specification with ValidatedMatchers {
+  "MiniBreakthroughTroyka initial fen" should {
+    val fen    = MiniBreakthroughTroyka.initialFen
+    val pieces = fen.toString()
+
+    "be valid" in {
+      pieces must_== "ppppp/ppppp/5/PPPPP/PPPPP w - - 0 1"
```

Comment:
this isnt really a test. this just checks you hardcoded the correct string in `src/main/scala/fairysf/variant/MiniBreakthroughTroyka.scala`
Maybe look to utilise `fairysf.format.Forysth.<<@` which converts a FEN into a Situation.
And then tests initialfen and a custom position